sirepo/template/radia.py

```diff
@@ -546,6 +565,11 @@ def _copy_geom_obj(o):
     return o_copy
 
 
+def _delim_string(val=None, default_val=None):
+    d = default_val if default_val is not None else []
+    return sirepo.util.to_comma_delimited_string(val if val is not None else d)
```

Comment:
I think this could all be simplified to 

`return sirepo.util.to_comma_delimited_string(val or default_val or [])`

Reply:
That does not work because you get ```ValueError: The truth value of an array with more than one element is ambiguous```